src/App.js

```diff
@@ -1,12 +1,138 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
+import Button from "./components/Button";
+import ArticleFruit from "./components/ArticleFruit";
+import {useForm} from 'react-hook-form';
+import FormInput from "./components/FormInput";
 
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
-    </>
-  );
+
+    const [amountStrawberry, setAmountStrawberry] = useState(0);
+    const [amountBanana, setAmountBanana] = useState(0);
+    const [amountApple, setAmountApple] = useState(0);
+    const [amountKiwi, setAmountKiwi] = useState(0);
+
+    const {register, handleSubmit, formState: { errors }} = useForm();
+
+    function handleFormSubmit(data) {
+        // geen preventDefault meer nodig, dit regelt Hook Form
+        console.log(`Aardbeien: ${amountStrawberry} \nBananen: ${amountBanana} \nAppels: ${amountApple} \nKiwi's: ${amountKiwi} \n` , data);
+    }
+
+    return (
+        <>
+            <header>
+                {/*<h1>Fruitmand bezorgservice</h1>*/}
+            </header>
+            <main>
+                <section className="container">
+                    <ArticleFruit
+                        articleTitle="🍓 Aardbeien"
+                        state={amountStrawberry}
+                        setState={setAmountStrawberry}
+                    />
+                    <ArticleFruit
+                        articleTitle="🍌 Bananen"
+                        state={amountBanana}
+                        setState={setAmountBanana}
+                    />
+                    <ArticleFruit
+                        articleTitle="🍏 Appels"
+                        state={amountApple}
+                        setState={setAmountApple}
+                    />
+                    <ArticleFruit
+                        articleTitle="🥝 Kiwi's"
+                        state={amountKiwi}
+                        setState={setAmountKiwi}
+                    />
+                    <Button
+                        buttonType="button"
+                        buttonClass="button"
+                        handleClick={() => setAmountStrawberry(0) + setAmountApple(0) + setAmountBanana(0) + setAmountKiwi(0)}
```

Comment:
Het is wellicht voor jezelf prettiger om hier even een aparte functie voor te maken:
```
const reset = () => {
    setAmountStrawberry(0);
    setAmountApple(0);
    setAmountBanana(0);
    setAmountKiwi(0);
}
```